core/management/commands/add_promo_logo.py

```diff
@@ -0,0 +1,49 @@
+import pathlib
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from PIL import Image, UnidentifiedImageError
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("number", type=int)
+        parser.add_argument("path", type=pathlib.Path)
+        parser.add_argument("-f", "--force", action="store_true")
+
+    def handle(self, number: int, path: pathlib.Path, force: int, *args, **options):
+        if not path.exists() or path.is_dir():
+            self.stderr.write(f"{path} is not a file or does not exist")
+            return
+
+        dest_path = (
+            pathlib.Path(apps.get_app_config("core").path)
+            / "static"
+            / "core"
+            / "img"
+            / f"promo_{number}.png"
+        )
+
+        if dest_path.exists() and not force:
+            over = input("File already exists, do you want to overwrite it? (y/N):")
+            if over.lower() != "y":
+                self.stdout.write("exiting")
+                return
+        try:
+            im = Image.open(path)
+            try:
+                im.resize((120, 120)).save(dest_path, format="PNG")
```

Comment:
```suggestion
                im.resize((120, 120), resample=Resampling.LANCZOS).save(dest_path, format="PNG")
```
Par défaut, Pillow utilise le filtre `NEAREST`, qui est plus performant, mais donne un moins bon résultat. Le filtre `LANCZOS` donne une meilleure qualité à l'image compressée.

Et c'est raccord avec le reste de la base de code, où on utilise toujours le filtre lanczos.